napari/layers/labels/_labels_key_bindings.py

```diff
@@ -74,7 +75,15 @@ def activate_labels_erase_mode(layer: Labels):
 )
 def new_label(layer: Labels):
     """Set the currently selected label to the largest used label plus one."""
-    layer.selected_label = np.max(layer.data) + 1
+    new_selected_label = np.max(layer.data) + 1
+    if layer.selected_label == new_selected_label:
+        show_info(
+            trans._(
+                "Largest label already selected",
```

Comment:
It sounds misinformation to me. Largest may mean Largest by volume/area. 
And here we select first empty labels num.

Reply:
Maybe the message could be then something like `Maximum label value plus one already set as current selected label` or something like that which explains the situation with a more specific/correct wording? Not totally sure what words will describe better that `layer.selected_label` is already equal to `np.max(layer.data) + 1` 🤔

Reply:
This proposition is much better.

Reply:
Yeah it's tricky.
`Maximum label value plus one already set as current selected label` isn't exactly intuitive either.
maybe `highest unused label`?

Reply:
So something like `Highest unused label already set as current selected label`?

Reply:
No. Higest unused often is `np.iinfo(dtype).max`

maybe 

`Highest used label plus one already set as current selected label`

Reply:
I'd say that's an implementation detail?

Reply:
`higest used plus 1` will mean `lower unused` in 99% of the time. But if there is any hole in used labels, then it should land in this hole.

Reply:
Depends which way you look at it.
You're saying lowest of the labels that are unused, so from the PoV of the extant labels.
I'm looking from PoV of the user and the widget, where if you've used 1, 2, 3, then 4 is highest unused.

Reply:
In the end we need a message -- notification -- that is targeted at a user painting labels and is clear as to why seemingly nothing happened when they pushed the keybind.

Reply:
Maybe something like `Current selected label is not being used. You will need to use it to be able to set the current select label to the next label available` ? Kind of wordy but maybe can sound like more aimed to an end user which is unaware of implementation details but still wants to increase the current selected label? Could it even be worthy to mention in the notification that to increase the label regardless of it being used you can use the `=` shortcut?

Reply:
I like that!

Reply:
What do you think about structuring the message [in that way](https://github.com/napari/napari/pull/6546#discussion_r1432857185) @Czaki ?

Reply:
looks ok